eng/common/scripts/Create-APIReview.ps1

```diff
@@ -81,23 +82,47 @@ else
 }
 
 $FoundFailure = $False
+$pkgInfoPath = Join-Path -Path $ArtifactPath "PackageInfo"
 foreach ($pkgName in $responses.Keys)
 {    
     $respCode = $responses[$pkgName]
     if ($respCode -ne '200')
     {
-        $FoundFailure = $True
-        if ($respCode -eq '201')
+        $pkgPropPath = Join-Path -Path $pkgInfoPath ($PackageName + ".json")
+        if (-Not (Test-Path $pkgPropPath))
         {
-            Write-Host "API Review is pending for package $pkgName"
+            Write-Host " Package property file path $($pkgPropPath) is invalid."
+            $FoundFailure = $True
         }
         else
         {
-            Write-Host "Failed to create API Review for package $pkgName"
+            $pkgInfo = Get-Content $pkgPropPath | ConvertFrom-Json
+            $version = [AzureEngSemanticVersion]::ParseVersionString($pkgInfo.Version)
+            if ($version.IsPrerelease)
+            {
+                Write-Host "Package version is not GA. Ignoring API view approval status"
+            }
+            elseif ($pkgInfo.SdkType -eq "management")
```

Comment:
What about track 1 libraries? Perhaps we should change this to be sdkType = client instead.

Reply:
I would like to discuss about this more.